spec/jobs/education_form/delete_old_applications_spec.rb

```diff
@@ -5,12 +5,13 @@
   before do
     @claim_nil = create(:education_benefits_claim, processed_at: nil)
     @claim_new = create(:education_benefits_claim, processed_at: Time.now.utc)
-    @claim_old = create(:education_benefits_claim, processed_at: 2.months.ago)
+    @claim_new = create(:education_benefits_claim, processed_at: 45.days.ago.utc)
```

Comment:
Does this instance variable get used anywhere? It used to refer to the claim being created at `Time.now.utc`, but it will be set to the one created 45 days ago. If not, can this not be assigned? I find it confusing to have the variable set in one line, then overwritten in the next